cloudsmith_cli/data/VERSION

```diff
@@ -1 +1 @@
-0.3.1
+0.3.2
```

Comment:
Probably better not to include the version bump in the PR - We release it separately via `bumpversion` so that it atomically bumps the version and tags that specific commit. We check for that on travis in order to automate upload to Cloudsmith and PyPi. :-)